### DISCUSSION TITLE
STL files for the cover of the 2024 Badge?

### DISCUSSION THREAD

**Discussion by syntaqx** [badger/home#5] [Ideas]
Would you be able to provide, or add to the repository the STL files (or equivalent) for the battery case? We weren't able to grab some one today, and could just print our own.

**Comment by jradwan** [comment]
I found the STL here: https://github.com/badger/home/blob/main/badger-back.stl

![image](https://github.com/user-attachments/assets/f90eebc3-9231-497a-a4f5-b841d56d6c53)
